cmake/AddResources.cmake

```diff
@@ -61,6 +68,14 @@ function(add_resources target)
 				"${android_dir}/assets/${file}"
 				SYMBOLIC
 			)
+		# Assets will already be installed by MACOSX_PACKAGE_LOCATION on macOS
+		elseif(NOT APPLE)
+			# No trailing slash is required or else it will strip the last path
+			# of the directory
+			install(FILES "assets/${file}"
+				DESTINATION "${INSTALL_APPDATADIR}${file_dir}"
+				COMPONENT "${target}"
```

Comment:
Not directly related but probably should just delete the `COMPONENT` from everywhere; that was supposed to make it possible for the MSI installer to have choosable options for the client, the stand-alone server, and the command-line tools, but I couldn’t get CPack to correctly generate a non-monolithic installer (IIRC it just ran the `install(CODE)` whenever it felt like it no matter what I did) so now these are all just superfluous.

Reply:
Removed 'em.